checks/grouping.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+from astropy.table import Table
+from astropy.table import Table
+from astropy.coordinates import Angle, SkyCoord
+from gammapy.data import DataStore
+from gammapy.data import ObservationGroupAxis
+from gammapy.data import ObservationGroups
+from gammapy.datasets import gammapy_extra
+import pylab as pt
+pt.ion()
+
+dir=str(gammapy_extra.dir)+'/datasets/hess-crab4'
+data_store = DataStore.from_dir(dir)
+Observation_Table=data_store.obs_table
+alt = Angle([0, 30, 60, 90], 'degree')
+az = Angle([-90, 90, 270], 'degree')
+ntels = np.array([3, 4])
+list_obs_group_axis = [ObservationGroupAxis('ALT', alt, 'bin_edges'),
+                       ObservationGroupAxis('AZ', az, 'bin_edges'),
+                       ObservationGroupAxis('N_TELS', ntels, 'bin_values')]
+obs_groups = ObservationGroups(list_obs_group_axis)
+obs_table_grouped = obs_groups.group_observation_table(Observation_Table)
```

Comment:
I get this error:
´´´
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
/Users/jouvin/Desktop/these/test_Gammapy/gammapy-extra/datasets/grouping.py in <module>()
     22                        ObservationGroupAxis('N_TELS', ntels, 'bin_values')]
     23 obs_groups = ObservationGroups(list_obs_group_axis)
---> 24 obs_table_grouped = obs_groups.group_observation_table(Observation_Table)
     25 print(obs_table_grouped)

/Users/jouvin/gammapy/gammapy/data/observation.pyc in group_observation_table(self, obs_table)
    795                 selection = dict(type='par_box', variable=name,
    796                                  value_range=(min_value, max_value))
--> 797                 obs_table_selected = obs_table_selected.select_observations(selection)
    798             # define group and fill in list of grouped observation tables
    799             group_id_data = i_group * np.ones(len(obs_table_selected), dtype=np.int)

/Users/jouvin/gammapy/gammapy/data/observation.pyc in select_observations(self, selection)
    391                 obs_table = obs_table.select_range(selection['variable'],
    392                                                    selection['value_range'],
--> 393                                                    selection['inverted'])
    394 
    395             else:

/Users/jouvin/gammapy/gammapy/data/observation.pyc in select_range(self, selection_variable, value_range, inverted)
    191         # check that variable exists on the table
    192         if selection_variable not in obs_table.keys():
--> 193             raise KeyError('Key not present in table: {}'.format(selection_variable))
    194 
    195         value_range = Quantity(value_range)

KeyError: u'Key not present in table: ALT'

´´´